package.json

```diff
@@ -104,6 +104,9 @@
         "react-dom": "18",
         "eslint": "9"
       }
+    },
+    "overrides": {
+      "@joshwooding/vite-plugin-react-docgen-typescript": "0.5.0"
```

Comment:
This is a temporary fix until storybook updates the dependency in their react-vite framework package.